examples/simple.py

```diff
@@ -3,14 +3,17 @@
 import time
 
 from saq import CronJob, Queue
+from saq.types import Context, SettingsDict
 
+CtxType = Context
```

Comment:
doesn't seem like this alias is necessary, let's remove it?

Reply:
> doesn't seem like this alias is necessary, let's remove it?

You're right, totally unnecessary. Also don't need CtxType, we just use vanilla base `Context` in this example